wideq/ac.py

```diff
@@ -4,6 +4,28 @@
 
 from .client import Device
 
+class ACVSwingMode(enum.Enum):
+    """The vertical swing mode for an AC/HVAC device."""
+    OFF = "@OFF"
```

Comment:
Blades are numbered vertically from 1 (topmost) to 6.

Reply:
Cool! Maybe it would be nice to include this information in a comment or the docstring for these enums?